pkgs/applications/misc/keepassx/2.0-http.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, libgcrypt, qt5, zlib, libmicrohttpd, xorg, ... }:
```

Comment:
Those ```...``` probably shouldn't be there. I'm also guessing pulling all of xorg might be too much...